recipe/meta.yaml

```diff
@@ -1,14 +1,14 @@
 {% set name = "pypy3.6" %}
-{% set version = "7.1.0b" %}
+{% set version = "7.1.1b" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
-  - url: https://bitbucket.org/pypy/pypy/downloads/pypy3.6-v7.1.0-src.tar.bz2
-    fn: pypy3.6-v7.1.0-src.tar.bz2
-    sha256: faa81f469bb2a7cbd22c64f22d4b4ddc5a1f7c798d43b7919b629b932f9b1c6f
+  - url: https://bitbucket.org/pypy/pypy/downloads/pypy3.6-v7.1.1-src.tar.bz2
+    fn: pypy3.6-v7.1.1-src.tar.bz2
```

Comment:
You can drop the `fn` field and it would be nice to use the `jinja` variable for the version in the URL, that way the bot can send a PR updating it. The only problem here seems to be that the `b` is not in the file/URL version :-/